scripts/changeset/changeset-version-with-docs.ts

```diff
@@ -7,21 +7,33 @@ import { error } from 'console';
   try {
     /**
      * This is the base command that has to run always.
-     * Release CIs were failing when there were only empty changesets because we weren't running this command.
-     * See more here: https://github.com/FuelLabs/fuels-ts/pull/1847
+     *
+     * Release CIs were failing when there were only empty changesets
+     * because we weren't running this command.
+     *
+     * See more here:
+     *  https://github.com/FuelLabs/fuels-ts/pull/1847
      */
     execSync(`changeset version`);
 
     // Invoke versions' prebuild script (will rewrite version files if needed)
     execSync(`pnpm -C packages/versions prebuild`);
 
+    // Invoke fuels' build:proxy script (will rewrite header versions in generated files)
+    execSync(`pnpm -C packages/fuels build:proxy`);
+
+    // Checks if the commands above generated git changes
     const versionsFilePath = `packages/versions/src/lib/getBuiltinVersions.ts`;
+    const proxyDirPath = `packages/fuels/src/cli/commands/deploy/proxy`;
 
     const versionsChanged = !!execSync(`git status --porcelain ${versionsFilePath}`)
       .toString()
       .trim();
 
-    if (versionsChanged) {
+    const proxyChanged = !!execSync(`git status --porcelain ${proxyDirPath}`).toString().trim();
+
+    // If they did, add and commit the changes
+    if (versionsChanged || proxyChanged) {
```

Comment:
I think that we should add the whole `proxyDirPath` if it has changed:
```ts
if (versionsChanged || proxyChanged) {
      if(versionsChanged) execSync(`git add ${versionsFilePath}`);
      if(proxyChanged) execSync(`git add ${proxyDirPath}`);
      execSync(`git commit -m"ci(scripts): update versions"`);
}
```

Reply:
Great catch! 0c34b7912f64b095f4989caf2ff0708c4805552b